epic_issues.go

```diff
@@ -0,0 +1,121 @@
+package gitlab
+
+import "fmt"
+
+// ListEpicIssues get a list of epic issues.  This function accepts
+// pagination parameters page and per_page to return the list of project issues
+//
+// Gitlab API docs: https://docs.gitlab.com/ee/api/epic_issues.html#list-issues-for-an-epic
+func (s *EpicsService) ListEpicIssues(gid interface{}, epic int, opt *ListOptions, options ...RequestOptionFunc) ([]*Issue, *Response, error) {
+	group, err := parseID(gid)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	u := fmt.Sprintf("groups/%s/epics/%d/issues", group, epic)
+
+	req, err := s.client.NewRequest("GET", u, opt, options)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	var i []*Issue
+	resp, err := s.client.Do(req, &i)
+	if err != nil {
+		return nil, resp, err
+	}
+
+	return i, resp, err
+}
+
+// EpicIssueAssignment contains both the Epic and Issue objects returned from Gitlab w/ the assignment ID
+type EpicIssueAssignment struct {
+	ID    int    `json:"id"`
+	Epic  *Epic  `json:"epic"`
+	Issue *Issue `json:"issue"`
+}
+
+// AssignEpicIssue assigns an existing issue to an Epic
+//
+// Gitlab API Docs: https://docs.gitlab.com/ee/api/epic_issues.html#assign-an-issue-to-the-epic
+func (s *EpicsService) AssignEpicIssue(gid interface{}, epic int, issue int, options ...RequestOptionFunc) (*EpicIssueAssignment, *Response, error) {
```

Comment:
When declaring two params of the same type, you can write that a bit nicer/shorter:
```suggestion
func (s *EpicsService) AssignEpicIssue(gid interface{}, epic, issue int, options ...RequestOptionFunc) (*EpicIssueAssignment, *Response, error) {
```